### DISCUSSION TITLE
several issues with daterange

### DISCUSSION THREAD

**Discussion by Code-Peter** [easepick/easepick#169] [Q&A]
Hello, I am using daterange and have  some difficulties in getting it working as desired. 

1) Showing two calendars side-by-side
I added grid and calendars to both LockPlugins and to RangePlugins, but it only shows 1 calendar. 

2) Limit years 
I thought I could add yearRange: "-0:+1". 
Cannot get it to work

3) showing two input fields for daterange
I do not manage to show two fields for first and second date. It only shows one by default. 

Any help? Thanks

**Comment by megasmack** [comment]
1. To set two calendars side-by-side, you'll need to set `calendars` and `grid` in the config options.
```javascript
{
     grid: 2,
     calendars: 2
}
```

2.  You'll need to enable the AmpPlugin and pass a configuration. You could do some date math to dynamically set the year min and max. Some may like this:
```javascript
const currentDate = new Date();
const currentYear = currentDate.getFullYear();
const previousYear = currentYear - 1;
const nextYear = currentYear + 1;
```
   Then add those properties to you config object.

   ```javascript
   {
     AmpPlugin: {
       dropdown: {
         years: true,
         minYear: previousYear,
         maxYear: nextYear
       }
     }
   }
   ```

3. Using the RangePlugin, you'll want to define `elementEnd` and give it the element of your end date input.
```javascript
{
    RangePlugin: {
       elementEnd: "#checkout"
    }
}
```
